src/test/kotlin/com/fasterxml/jackson/module/kotlin/KotlinModuleTest.kt

```diff
@@ -0,0 +1,67 @@
+package com.fasterxml.jackson.module.kotlin
+
+import org.junit.Assert.*
+import org.junit.Test
+
+class KotlinModuleTest {
```

Comment:
Test because mis-wiring one of these booleans would be easy to do.

Reply:
Good idea as well.